sklearn/multiclass.py

```diff
@@ -476,11 +477,12 @@ def coef_(self):
 
     # TODO: Remove intercept_ attribute in 0.26
     # mypy error: Decorated property not supported
-    @deprecated("Attribute intercept_ was deprecated in "  # type: ignore
-                "version 0.24 and will be removed in 0.26. "
-                "If you observe this warning while using RFE "
-                "or SelectFromModel, use the importance_getter "
-                "parameter instead.")
+    @deprecated(  # type: ignore
```

Comment:
There's already a comment for `mypy` errors just above. Can we remove one of them or do they address different things?

Reply:
We can remove one of them. Although I do not mine the verbosity here.

Side note: I think it would be nice to have a `_deprecated_property` that just does the correct thing regarding order + mypy. (If possible)